user/reference/bionic.md

```diff
@@ -107,13 +107,23 @@ For preinstalled language interpreters, a standard version manager like `rvm` is
 
 ## Python support
 
-* Images for all languages have Python 2 and 3 installations provided by the distribution. The Python 2 one has pip preinstalled.
+* Images for all languages have Python 2 (`2.7.17`) and 3 (`3.6.9`) installations provided by the distribution. The Python 2 one is the default and has pip preinstalled.
```

Comment:
Do we sill want to default to Python 2 now that it is end of life?
* https://devguide.python.org/#status-of-python-branches vs.
* https://devguide.python.org/devcycle/#end-of-life-branches

Reply:
It's not "we", it's the distribution. That's what you end up with when ou install `python` and `python3` Apt packages.

They are likely to switch to Python 3 as the default in the next LTS release.

Reply:
The very last and truely final Python 2.7.18 is in final candidate and is scheduled to be released tomorrow.

Reply:
> That's what you end up with when you install python and python3 Apt packages.

Can we not install `apt install python-pip` and instead `apt install python3-pip`?  End of life is end of life.

Reply:
Actually, [Pip is installed via `get-pip.py`](https://github.com/travis-ci/travis-cookbooks/blob/b5e4fc1f3cd153f9a8485998eed63789e7c146b1/cookbooks/travis_build_environment/recipes/pip.rb) -- presumably, to get the latest version as of build image creation time rather than an older one that the distribution provides.

Create a [feature request](https://travis-ci.community/c/product/feature-requests/17) and/or a pull request against https://github.com/travis-ci/travis-cookbooks if you have a workable improvement.

Reply:
travis-ci/travis-cookbooks#1076

Reply:
As discussed above [[RELEASE] Python 2.7.18, the end of an era[(https://mail.python.org/archives/list/python-dev@python.org/message/OFCIETIXLX34X7FVK5B5WPZH22HXV342/) 
* https://pythoninsider.blogspot.com/2020/04/python-2718-last-release-of-python-2.html
* https://www.python.org/downloads/release/python-2718/

Reply:
"Default" here solely means that it's available as `python` -- which it is.

I'm fine with not mentioning this explictly -- I only did because the new text that I was merging was saying that.